### PR TITLE
Cw/wrap widths

### PR DESCRIPTION
**Pull request by chriswynn**

### CODE REVIEW

File: src/components/Column/index.tsx [hunk 1]
```diff
@@ -5,7 +5,7 @@ import { Theme } from "types/global";
 const Column: React.FC = props => (
   <div
     css={({ ...theme }: Theme) => ({
-      flexBasis: "calc((42rem - 100%) * 9999)",
+      flexBasis: "calc((30rem - 100%) * 9999)",
```
Comment (jssee) [inline src/components/Column/index.tsx]: should this value be put into the theme file?

File: src/lib/utils.ts [hunk 2]
```diff
@@ -1,3 +1,7 @@
 import facepaint from "facepaint";
 
-export const mq = facepaint(["@media(min-width: 600px)"]);
+export const mq = facepaint([
+  "@media(min-width: 600px)",
```
Comment (jssee) [inline src/lib/utils.ts]: might be wrong but isnt this supposed to be `max-width` ?

File: src/components/Column/index.tsx (hunk 1 above)
Comment (chriswynn) [reply src/components/Column/index.tsx]: Possibly but it is only used in this one spot so not sure if it is necessary

File: src/types/global.d.ts [hunk 3]
```diff
@@ -20,7 +20,7 @@ export type Theme = {
       };
     };
   };
-  maxWidths: string[];
+  maxWidths: Array<string[]>;
```
Comment (sethkrasnianski) [inline src/types/global.d.ts]: Doesn't this commit render type errors until d7db82aaf384dd625527cf7af42a96c7490aa827 is applied?

**Pull request merged**